modules/deployments-kube-deployments.adoc

```diff
@@ -7,7 +7,7 @@
 
 Kubernetes provides a first-class, native API object type in {product-title} called `Deployment`. `Deployment` objects serve as a descendant of the {product-title}-specific `DeploymentConfig` object.
```

Comment:
Not sure what we mean and want to tell by this.

```
 `Deployment` objects serve as a descendant of the {product-title}-specific `DeploymentConfig` object.
```

Since we are deprecating the deployment configs, it is probably okay to remove this sentence.

Reply:
+1, will remove that sentence